b0realsim/label_to_chi.py

```diff
@@ -65,48 +49,8 @@ def label_to_chi(bids_subject_dir):
     bids_sidecar['input file'] = str(merged_labels_path.resolve())
     bids_sidecar['command'] = 'python label_to_chi.py -s ' + str(bids_subject_dir)
 
-    bids_sidecar['anatomy'] = {}
```

Comment:
Why remove the bids sidecar that report the values  that were used to generate the map from the label? I think this is valuable - especially since the values can change with new information, as you have discovered re: chi.

An alternative would *maybe* be to report the tissue2MR version / commit, but this makes it more difficult to actually go find the values / fetch them in the json sidecar files in a script.